base/array.jl

```diff
@@ -1070,11 +1096,12 @@ function push!(a::Vector{Any}, @nospecialize x)
     return a
 end
 function push!(a::Vector{Any}, @nospecialize x...)
+    @_terminates_locally_meta
     na = length(a)
     nx = length(x)
     _growend!(a, nx)
-    for i = 1:nx
-        arrayset(true, a, x[i], na+i)
+    @inbounds for i = 1:nx
```

Comment:
This does not seem to have any inlining?
```suggestion
    for i = 1:nx
```